src/common/utils/setup-helpers.js

```diff
@@ -388,13 +388,18 @@ function configureDAIDefaultOptions(options: KPOptionsObject): void {
 function configureBumperDefaultOptions(options: KPOptionsObject): void {
   const bumperPlugin = Utils.Object.getPropertyPath(options, 'plugins.bumper');
   const daiPlugin = Utils.Object.getPropertyPath(options, 'plugins.imadai');
-  if (bumperPlugin && !bumperPlugin.disable && daiPlugin && !daiPlugin.disable) {
+  if (bumperPlugin) {
+    const bumperConfig: Object = {};
+    if (isIos() && options.playback.playsinline === false) {
+      bumperConfig['playOnMainVideoTag'] = true;
```

Comment:
need to make sure application config takes precedence
Better add a test for this - we haven't done it for any of the setup helpers when come to think of it, so better start sometime, like now :-)